### PR TITLE
[RFR] Playbooks for templates

### PR DESCRIPTION
**Pull request by vatsalparekh**
Adding Support and examples for Template support from KubeVirt module

### CODE REVIEW

File: tests/playbooks/kubevirt_raw_template.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+- name: Create a Template
+  hosts: localhost
+  connection: local
+  vars:
+    ansible_python_interpreter: "/usr/bin/env python"
+  tasks:
+    - name: Create Template template-demo
+      kubevirt_raw:
+       state: absent
```
Comment (mmazur) [inline tests/playbooks/kubevirt_raw_template.yml]: Why absent?

**Comment by pkliczewski** [comment]
@vatsalparekh Is this ready once https://github.com/kubevirt/ansible-kubevirt-modules/pull/113 was merged?

**Comment by pkliczewski** [comment]
@mmazur review?

**Comment by vatsalparekh** [comment]
@pkliczewski yes this is ready for review/merge

**Comment by pkliczewski** [comment]
@machacekondra review?

File: tests/playbooks/kubevirt_facts_template.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+- name: Gather facts for Templates
+  hosts: localhost
+  connection: local
+  vars:
+    ansible_python_interpreter: "/usr/bin/env python"
```
Comment (machacekondra) [inline tests/playbooks/kubevirt_facts_template.yml]: What is the reason to use this? I think we should keep the playbook as simple as possible.

File: tests/playbooks/kubevirt_facts_template.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  tasks:
+    - name: facts for Template in default namespaces
+      k8s_facts:
+        kind: Template
+        name: working
+        namespace: default
+      register: template_fact
+    - debug:
```
Comment (machacekondra) [inline tests/playbooks/kubevirt_facts_template.yml]: I know these are just tests, but can we use some common practise with yaml formatting, like all tasks has names, do newlines between tasks etc?

**Comment by mmazur** [comment]
As far as the contents of the commits go, looks ok to me.

**Pull request merged**